storage/cloudspanner/sdl_test.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cloudspanner
+
+import "testing"
+
+func TestDDL(t *testing.T) {
+	_, err := readDDL()
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
Add comment that readDDL functionality is tested in map_storage_test.go.

Reply:
Testing DDL isn't really the primary purpose of `map_storage_test`, so if you don't mind I'll defer that.